examples/enroll/enroll.ino

```diff
@@ -84,8 +86,11 @@ void loop()                     // run over and over again
   }
   Serial.print("Enrolling ID #");
   Serial.println(id);
-
-  while (!  getFingerprintEnroll() );
+  //assume an error happened
+  int error = 1;
+  while (error){
+    error = getFingerprintEnroll();
+  }
```

Comment:
This is functionally equivalent. Why the change?

Reply:
It has been a long time. I think I changed it because it took me a long time to figure out what the original actually meant.

Reply:
OK, for now plz make the PR just the fix for the "." prints.

Reply:
Done.